docker/android-sdk/tools/watch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+inotifywait -mr $HOME/.android -e create -e moved_to -e modify -e attrib |
```

Comment:
some builds create/overwrite some files in .android folder but it messes up the permission in the shared pv so installing new android packages or building apps might fail because of wrong file permissions.